src/structures/interfaces/InteractionResponses.js

```diff
@@ -224,6 +226,53 @@ class InteractionResponses {
     return options.fetchReply ? this.fetchReply() : undefined;
   }
 
+  /**
+   * Shows a modal component
+   * @param {Modal|ModalOptions} modal The modal to show
+   * @returns {Promise<void>}
+   */
+  async showModal(modal) {
+    const _modal = modal instanceof Modal ? modal : new Modal(modal);
+    await this.client.api.interactions(this.id, this.token).callback.post({
+      data: {
+        type: InteractionResponseTypes.MODAL,
+        data: _modal.toJSON(),
+      },
+    });
+  }
+
+  /**
+   * An object containing the same properties as CollectorOptions, but a few more:
```

Comment:
This doesn't look like a few more, but a few less (did you forget to extend)? Additionally, should these have options allow for listening to a specific channel or message like [`InteractionCollectorOptions`](https://discord.js.org/#/docs/discord.js/main/typedef/InteractionCollectorOptions) does?

Reply:
I think this is just missing from the JSDoc, from me copy pasting. I'll make sure it matches the typings which iirc do extend and support that

Reply:
Bad wording but largely no, I don't think this should have more options. This is the promisified version tied to a specific interaction - I could internally pass `interaction.channel` and `interaction.message` but they shouldn't be set by the user.

If they want to do that, use `InteractionCollector`